pytorch_lightning/callbacks/lr_monitor.py

```diff
@@ -155,11 +155,26 @@ def _extract_stats(self, trainer, interval: str) -> Dict[str, float]:
 
         return latest_stat
 
-    def _extract_lr(self, param_group, name: str) -> Dict[str, float]:
+    def _extract_lr(self, trainer, param_group, name: str) -> Dict[str, float]:
         lr = param_group.get('lr')
-        self.lrs[name].append(lr)
+        try:
+            self.lrs[name].append(lr)
+        except KeyError:
```

Comment:
it feels like `if name in self.lrs ... else ...` is better than try except but I could be wrong. maybe this is even slightly faster? ^^

Reply:
Thanks @awaelchli. Found a bug :)